frontend/src/data/ExtDataMine.ts

```diff
@@ -0,0 +1,253 @@
+export type FileType = {
+  ext: string;
+  mime: string;
+};
+
+export const extDataMine: FileType[] = [
+  { ext: "js", mime: "application/javascript" },
+  { ext: "ts", mime: "application/typescript" },
+  { ext: "jsx", mime: "application/javascript" },
+  { ext: "tsx", mime: "application/typescript" },
+  { ext: "json", mime: "application/json" },
+  { ext: "html", mime: "text/html" },
+  { ext: "css", mime: "text/css" },
+  { ext: "scss", mime: "text/x-scss" },
+  { ext: "sass", mime: "text/x-sass" },
+  { ext: "md", mime: "text/markdown" },
+  { ext: "xml", mime: "application/xml" },
+  { ext: "php", mime: "application/x-php" },
+  { ext: "rb", mime: "application/x-ruby" },
+  { ext: "py", mime: "application/x-python" },
+  { ext: "java", mime: "text/x-java-source" },
+  { ext: "kt", mime: "text/x-kotlin-source" },
+  { ext: "go", mime: "application/x-go-source" },
```

Comment:
Those are `mime` types, not file types. 

I update the mapping of file types in the backend here: https://github.com/visevol/Cscope/pull/119

^^ This should reduce the number of `unknown` files. 

I also added an API endpoint that returns the list of file types in the project. Example:
![image](https://github.com/user-attachments/assets/eccbf636-beee-4258-9f0b-ad8e8c124e02)

We already categorize source files in the backend, can we not do it again in the frontend? Just use the `filetype` on each source files you get from the API instead.

Reply:
It's delete